_security-plugin/authentication-backends/authc-index.md

```diff
@@ -1,23 +1,28 @@
 ---
 layout: default
-title: Authentication flow
-parent: Configuration
-nav_order: 1
+title: Authentication backends
+nav_order: 45
+has_children: true
+has_toc: false
+redirect_from:
+  - /security-plugin/authentication-backends/
 ---
 
-# Authentication flow
+# Authentication backends
 
-Understanding the authentication flow is a great way to get started with configuring the security plugin.
+Authentication backend configurations determine the method or methods you use for authenticating users and the way users pass their credentials and sign in to OpenSearch. Having an understanding of the basic authentication flow before getting started can help with the configuration process for whichever backend you choose. Consider the high-level sequence of events in the description that follows, and then refer to the detailed steps for configuring the authentication type you choose to use with OpenSearch.
 
-1. To identify a user who wants to access the cluster, the security plugin needs the user's credentials.
+## Authentication flow
+
+1. To identify a user who wants to access the cluster, the Security plugin needs the user's credentials.
 
    These credentials differ depending on how you've configured the plugin. For example, if you use basic authentication, the credentials are a user name and password. If you use a JSON web token, the credentials are stored within the token itself. If you use TLS certificates, the credentials are the distinguished name (DN) of the certificate.
 
-2. The security plugin authenticates the user's credentials against a backend: the internal user database, Lightweight Directory Access Protocol (LDAP), Active Directory, Kerberos, or JSON web tokens.
+2. The Security plugin authenticates the user's credentials against a backend: the internal user database (basic authentictation), Lightweight Directory Access Protocol (LDAP)/Active Directory, JSON web tokens, SAML, or another authentication protocol.
 
    The plugin supports chaining backends in `config/opensearch-security/config.yml`. If more than one backend is present, the plugin tries to authenticate the user sequentially against each until one succeeds. A common use case is to combine the internal user database of the security plugin with LDAP/Active Directory.
 
-3. After a backend verifies the user's credentials, the plugin collects any backend roles. These roles can be arbitrary strings in the internal user database, but in most cases, these backend roles come from LDAP/Active Directory.
+3. After a backend verifies the user's credentials, the plugin collects any backend roles. These roles can be arbitrary strings in the internal user database, roles retrieved from the LDAP/Active Directory server, or roles that are kept as attributes with the SAML protocol.
```

Comment:
Extraction of backend roles is applicable for all backends except for basic auth with the internal user database. LDAP is a bit exceptional because `authz` of the security plugin's `config.yml` needs to be configured to specify how roles should be extracted from the directory service. 

For the internal user database, roles are determined by roles that the user is mapped to within opensearch. With an external IdP, a cluster admin must map backend roles to roles in OpenSearch and users will transitively get OpenSearch roles based on the roles mapping. i.e. In LDAP I may be in the `My Org Developers` group, and in opensearch there could be a role named `developers` which has a certain set of permissions. The cluster admin would need to map `My Org Developers` to the `developers` role to map the external group to the internal role within OpenSearch.

Reply:
@cwperks I tried to sum this up while keeping it general.
`After a backend verifies the user's credentials, the plugin collects any [backend roles]({{site.url}}{{site.baseurl}}/security-plugin/access-control/index/#concepts). The authentication provider determines the way these roles are retrieved. For example, LDAP extracts backend roles from its directory service based on their mappings to roles in OpenSearch, while SAML stores the roles as attributes. When basic authentication is used, the internal user database refers to role mappings configured in OpenSearch.`

Reply:
Sounds good to me